code/modules/lootpanel/contents.dm

```diff
@@ -18,6 +18,8 @@
 
 	for(var/atom/thing as anything in source_turf.contents)
 		// validate
+		if(!istype(thing))
+			continue
```

Comment:
There should never be a non-atom inside of a turfs contents. This needs to be a stack trace